### PR TITLE
fix: update-roadmap

### PR DESCRIPTION
**Pull request by gabs-local**
task: local-766

### CODE REVIEW

File: src/pages/Home.vue [hunk 1]
```diff
@@ -22,10 +22,7 @@
         </p>
         <div class="wrap-cta">
           <p>Want to know more about us?</p>
-          <a
-            href="https://localp2p.money/local-litepaper.pdf"
-            target="_blank"
-          >
+          <a href="https://localp2p.money/local-litepaper.pdf" target="_blank">
```
Comment (davirds) [inline src/pages/Home.vue:25]: We should change this url from https://localp2p.money/local-litepaper.pdf to https://localmoney.io/local-litepaper.pdf

**Review by davirds — CHANGES_REQUESTED** [review]

File: src/pages/Home.vue (hunk 1 above)
Comment (gabs-local) [reply src/pages/Home.vue]: Done.

**Comment by gabs-local** [comment]
Litepaper url updated

**Review by davirds — APPROVED** [review]
LGTM!

**Pull request merged**